crates/cli/src/args.rs

```diff
@@ -260,9 +261,26 @@ pub struct Args {
     /// Event signature for log decoding
     #[arg(long, value_name = "tracer", help_heading = "Dataset-specific Options")]
     pub js_tracer: Option<String>,
+    /// Keep raw decoded columns like topic0, topic1, etc.
+    #[arg(long, help_heading = "Output Options")]
+    pub keep_raw_decoded: bool,
```

Comment:
wrong PR?

Reply:
> wrong PR?

hmmmm i'm not sure why this is here, i don't remember have written this, sorry  btw